src/components/authentication/UserDashboard.js

```diff
@@ -27,11 +28,17 @@ class RegularUser extends Component {
       <BrowserRouter>
         <Navbar {...this.props} />
         <Switch>
+          <Route exact path="/signin">

```

Comment:
Line 31-33 is what I meant by cheeky way. Same thing happens as the last comment.

The thing I wanted to achieve is here that once a regular user signs in, the first thing they will see is the annotations (assigned to them) page. Will ask you how to achieve it in a better way tomorrow.